src/elm/Page/Community.elm

```diff
@@ -1005,14 +1005,12 @@ viewTransfer loggedIn model transfer =
                 |> I18Next.tr loggedIn.shared.translations I18Next.Curly "transfer.info"
     in
     div [ class "border-b last:border-b-0 border-gray-500 flex flex-wrap items-start p-4" ]
-        [ div [ class "flex justify-between w-full" ]
-            [ p [] [ text (transferInfo transfer.from transfer.value transfer.to) ]
-            , p [ class "text-sm text-orange-500" ]
-                (Page.viewDateDistance
-                    (Utils.posixDateTime (Just transfer.blockTime))
-                    model.date
-                )
-            ]
+        [ p [ class "w-3/4" ] [ text (transferInfo transfer.fromId transfer.value transfer.toId) ]
+        , p [ class "w-1/4 text-sm text-orange-500" ]
+            (Page.viewDateDistance
+                (Utils.posixDateTime (Just transfer.blockTime))
+                model.date
+            )
```

Comment:
@Pancakem  this looks like a merge issue, but the deleted part is actually the one we should keep!

Reply:
Lemme put that back on.